### PR TITLE
Try to fix attaching gdb 2

### PR DESCRIPTION
**Pull request by tavplubix**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Detailed description / Documentation draft:
#32448 did not help

### CODE REVIEW

**Comment by tavplubix** [comment]
Stress test -  Found signal in gdb.log - `Thread 1 "clickhouse-serv" received signal SIGSTOP, Stopped (signal).` - signal was sent by `SYSTEM SUSPEND` query from test `01643_system)suspend`
I tried to add
```
handle SIGSTOP nostop noprint pass
handle SIGCONT nostop noprint pass
```
to `script.gdb`, but gdb hangs. Let's move this test to Integration tests.

File: docker/test/fuzzer/run-fuzzer.sh [hunk 1]
```diff
@@ -155,17 +155,34 @@ function fuzz
...
+handle SIGQUIT nostop noprint pass
+handle SIGPIPE nostop noprint pass
+handle SIGTERM nostop noprint pass
+handle SIGUSR1 nostop noprint pass
+handle SIGUSR2 nostop noprint pass
+handle SIG$RTMIN nostop noprint pass
+info signals
 continue
+backtrace full
+info locals
```
Comment (azat) [inline docker/test/fuzzer/run-fuzzer.sh:177]: But `backtrace full` should print local variables, why `info locals` is required?

Comment (azat) [reply docker/test/fuzzer/run-fuzzer.sh:177]: @tavplubix please see #34517

Comment (tavplubix) [reply docker/test/fuzzer/run-fuzzer.sh:177]: > But backtrace full should print local variables, why info locals is required?

It's not required actually

**Pull request merged**